sms/v1/openapi.yaml

```diff
@@ -1439,10 +1561,9 @@ components:
           items:
             $ref: "#/components/schemas/RestMessagePart"
         overallPrice:
-          type: string
+          type: number
           description: The overall sum of prices for all parts of this message
-          format: string
-          example: "0.01"
+          example: 0.01
```

Comment:
Could not test it. Is this correct?

Reply:
Nope was correct. Price is `string`

Reply:
OK, I've dropped the commit a1b0747eaaa182f4af921a9bad861890ebd69e53 .